### DISCUSSION TITLE
EMu Interface updates / cheatsheet for v9 / post v5.1

### DISCUSSION THREAD

**Discussion by magpiedin** [fieldmuseum/EMu-Documentation#1138] [FAQs]
Resources to navigate the interface for EMu v9 

From the Heather Curtis / USHMM:
- [EMu v6 How-to's](https://docs.google.com/document/d/1qUEQq4ObE1ZZwZ-OWAaMSzdWlx4OcYif/) ~ google doc
- [EMu v6 icons](https://docs.google.com/document/d/1PM5oTkWu78UC51qNFgFuzQdOjGjdel6l/) ~ google doc

**Comment by rondlg** [comment]
Note v6 and v9 look the same
